application/templates/cms_autosave/macros.html

```diff
@@ -155,11 +155,11 @@ <h1 class="preview heading-large{% if not field.data %} empty{% endif %}" data-e
 {% macro editable_link(link_url_field, link_text_field=None) -%}
 
     <div class="preview-and-edit">
-        <p class="preview pretend-link" data-empty-text="{{ link_url_field.label.text }}" tabindex={{ kwargs['tab_index'] }}>
+        <p class="preview pretend-link {% if not link_text_field.data %}empty{% endif %}" data-empty-text="{{ kwargs['empty_message'] }}" tabindex={{ kwargs['tab_index'] }}>
             {% if link_text_field and link_text_field.data -%}
                 {{ link_text_field.data }}
             {% else %}
-                {{ link_url_field.label.text }}
+                {{ kwargs['empty_message'] }}
```

Comment:
This breaks the "preview" style for Quality and methodology information - is that deliberate?